x-pack/solutions/observability/plugins/apm/public/components/shared/trace_waterfall/bar_details.tsx

```diff
@@ -34,10 +35,15 @@ export function BarDetails({
   const theme = useEuiTheme();
   const { getRelatedErrorsHref } = useTraceWaterfallContext();
 
-  const viewRelatedErrorsLabel = i18n.translate(
-    'xpack.apm.waterfall.embeddableRelatedErrors.unifedErrorCount',
-    { defaultMessage: 'View related errors' }
-  );
+  const viewRelatedErrorsLabel =
+    item.errorCount > 1
+      ? i18n.translate('xpack.apm.waterfall.embeddableRelatedErrors.unifedErrorCount', {
+          defaultMessage: 'View {count} related errors',
+          values: { count: item.errorCount },
+        })
+      : i18n.translate('xpack.apm.waterfall.embeddableRelatedErrors.unifedErrorCount.default', {
+          defaultMessage: 'View related error',
+        });
```

Comment:
Can you see if this works, please?
```suggestion
  const viewRelatedErrorsLabel = i18n.translate('xpack.apm.waterfall.embeddableRelatedErrors.unifedErrorCount', {
            defaultMessage: '{count, plural, one {View related error} other {View # related errors}}',
            values: {
              count: item.errorCount,
            },
          })
```